gfx/video_filters/Blargg_NTSC_SNES_Custom_pseudo_PCE_PSX_SNES_S-Video.filt

```diff
@@ -1,3 +1,7 @@
+# This filter preset was created by Cyber for use with their shader presets.
```

Comment:
Thanks @hunterk. I didn't specify before but I'm a man so "his" might be more appropriate than "their". Thanks for your patience, support and understanding.